pkgs/tools/misc/debian-devscripts/default.nix

```diff
@@ -1,25 +1,35 @@
 {lib, stdenv, fetchurl, xz, dpkg
-, libxslt, docbook_xsl, makeWrapper
+, libxslt, docbook_xsl, makeWrapper, writeScript
 , python3Packages
-, perlPackages, curl, gnupg, diffutils
+, perlPackages, curl, gnupg, diffutils, nano, pkg-config, bash-completion, help2man
 , sendmailPath ? "/run/wrappers/bin/sendmail"
 }:
 
 let
   inherit (python3Packages) python setuptools;
+  sensible-editor = writeScript "sensible-editor" ''
+    #!/bin/sh
+    exec ''${EDITOR-${nano}/bin/nano} "$@"
+  '';
 in stdenv.mkDerivation rec {
-  version = "2.16.8";
+  version = "2.22.2";
   pname = "debian-devscripts";
 
   src = fetchurl {
     url = "mirror://debian/pool/main/d/devscripts/devscripts_${version}.tar.xz";
-    sha256 = "0xy1nvqrnifx46g8ch69pk31by0va6hn10wpi1fkrsrgncanjjh1";
+    hash = "sha256-Fflalt2JxqLS0gq0wy88pXCqiNvHj7sfP7fLwdSmUCs=";
   };
 
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ xz dpkg libxslt python setuptools curl gnupg diffutils ] ++
+  buildInputs = [ xz dpkg libxslt python setuptools curl gnupg diffutils pkg-config bash-completion help2man ] ++
```

Comment:
pkg-config in buildInputs? You are sure this is not executed at build time?

Reply:
Oh shoot, yeah that should probably be a nativeBuildInput. Will change.

Reply:
Done; this is ready now.